spec/models/game_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  describe 'populate the board' do
+    it 'should give us 32 pieces upon board population' do
+      g = FactoryGirl.create(:game)
```

Comment:
There's a lot of repetition with the game creations that could be eliminated by instead using a `let` statement to create/assign the game outside of your describe blocks.

This will also have an added benefit of speeding up your tests. As it is right now the game is being created 5 times, which means we have methods running upwards of 80 times as can be seen by the 80s to the right of the pawn creations in Coveralls - https://coveralls.io/builds/7087197/source?filename=app%2Fmodels%2Fgame.rb#L35